client/main.js

```diff
@@ -118,7 +118,9 @@ Array.from( document.getElementsByClassName( 'wcc-root' ) ).forEach( ( container
 
 			const enhancers = [
 				applyMiddleware( ...middlewares ),
-				window.devToolsExtension && window.devToolsExtension(),
+				window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__({
+					name: `wc/wcs/${routeClassName}`
```

Comment:
The `name` attribute helps identify the instance, between the many WP/WC ones
![Screen Shot 2020-11-25 at 2 30 33 PM](https://user-images.githubusercontent.com/273592/100278614-cbd81500-2f2a-11eb-87d0-1f83bbf54206.png)